pysnooper/tracer.py

```diff
@@ -322,7 +325,7 @@ def __exit__(self, exc_type, exc_value, exc_traceback):
         elapsed_time_string = pycompat.timedelta_format(duration)
         indent = ' ' * 4 * (thread_global.depth + 1)
         self.write(
-            '{indent}Elapsed time: {elapsed_time_string}'.format(**locals())
+            '\033[33m'+'{indent}Elapsed time: {elapsed_time_string}'+'\033[0m'.format(**locals())
```

Comment:
`format` doesn't work anymore. Just get rid of the `+`.

Colour must always be optional. Some terminals will not be able to support it so users should be able to turn it off. In particular Windows won't support this without colorama.